Themis/Views/Custom/ConditionalRedactedViewModifier.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ConditionalRedactedViewModifier.swift
+//  Themis
+//
+//  Created by Andreas Cselovszky on 23.12.22.
+//
+
+import SwiftUI
+
+struct ConditionalRedacted: ViewModifier {
+    let disabled: Bool
+    let reason: RedactionReasons
+
+    func body(content: Content) -> some View {
+        if disabled {
+            content
+        } else {
+            content
+                .redacted(reason: reason)
+        }
+    }
+}
+
+extension View {
+    func redacted(_ disabled: Bool, reason: RedactionReasons) -> some View {
+        modifier(ConditionalRedacted(disabled: disabled, reason: reason))
+    }
+}
```

Comment:
This view modifier just adds the disabled option to the redacted view modifier. It makes this nice placeholder view which can be used while loading. I played around with it to indicate loading, but didn't use it in the end. However I do think its very likely it will be used in the future and also that there is no harm in just keeping it.

Reply:
I think I would rather delete it now and copy-paste it from the old commit later. Having dead code in the repo isn't so great and I think most of the people in the team won't even remember / know that this exists and recreate it themselves in this case anyways.